applications/FluidDynamicsApplication/python_scripts/adjoint_fluid_analysis.py

```diff
@@ -16,179 +16,117 @@ class AdjointFluidAnalysis(AnalysisStage):
     '''Main script for adjoint sensitivity optimization in fluid dynamics simulations.'''
```

Comment:
this file is basically a copy of the FluidDynAnalysis

I think it could make sense to derive from it and only override the necessary fcts (from what I can see only the CreateSolver fct)

This is how we do it with StructuralMechanics and ContactStructuralMechanics